README.md

```diff
@@ -54,50 +47,72 @@ The 2010 census tracts are used (redrawn every 10 years)
 This is done for each area shown on the map.
 
 For shapefile:
+
 - Get shapefile for census tracts
 - Possibly prune the shapefile to only include census tracts
-- Process the shapefile to have geoids at census tract granularity
-	- To better understand geoids, see https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
-	- For processing, see /scripts
-This shapefile is (inner-) joined with the given state's census data
+- Process the shapefile to have geoids at census tract granularity - To better understand geoids, see https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html - For processing, see /scripts
+  This shapefile is (inner-) joined with the given state's census data
 
+##### old census
```

Comment:
go ahead and delete the old census data retrieval information.
Thank you for updating the instructions!